File: statistics-service/statistics-service/src/main/java/com/nurdor_project/statistics_service/dto/VolunteerDto.java
package com.nurdor_project.statistics_service.dto;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.io.Serializable;

@AllArgsConstructor
@Data
public class VolunteerDto implements Serializable {
    @JsonProperty("id")
    int id;
    @JsonProperty("name")
    String name;
    @JsonProperty("surname")
    String surname;
    @JsonProperty("address")
    String address;
    @JsonProperty("phoneNumber")
    String phoneNumber;
    @JsonProperty("email")
    String email;
    @JsonProperty("username")
    String username;
    @JsonProperty("password")
    String password;
    @JsonIgnore
    @JsonProperty("profilePicture")
    String profilePicture;
    @JsonProperty("nearestCity")
    String nearestCity;
    @JsonProperty("volunteerRole")
    Integer volunteerRole;
}

File: mobile-app/app/src/main/java/com/example/nurdor_volunteer_app_v3/dto/volunteerDto/VolunteerDto.kt
package com.example.nurdor_volunteer_app_v3.dto.volunteerDto

data class VolunteerDto(
    val id: Int, 
    val name: String,
    val surname: String,
    val address: String,
    val phoneNumber: String,
    val email: String,
    val username: String,
    val profilePicture: String,
    val nearestCity: String,
    val volunteerRole: Int,
)